test/fixtures/account_activities.yml

```diff
@@ -3,3 +3,33 @@ one:
   invoice: one
   bank_transaction: one
   created_at: <%= Time.zone.parse('2010-07-05 10:00') %>
+
+renew_cash_1:
```

Comment:
It's considered a good taste to name entities semantically, not just <something>_1 etc. Please, consider renaming.